pulp_rpm/app/tasks/publishing.py

```diff
@@ -103,10 +103,21 @@ def publish_artifacts(self, content):
         """
         published_artifacts = []
         for content_artifact in ContentArtifact.objects.filter(
-                content__in=content.exclude(pulp_type="rpm.repo_metadata_file").distinct()
+                content__in=content.exclude(
+                    pulp_type="rpm.repo_metadata_file"
+                ).distinct()
         ).iterator():
+            relative_path = (
+                os.path.join(
+                    "Packages",
+                    content_artifact.relative_path[0].lower(),
+                    content_artifact.relative_path
+                )
+                if content_artifact.content.pulp_type == "rpm.package"
+                else content_artifact.relative_path
+            )
```

Comment:
Subjective comment on readability:

`a = x if y else z` is great as a one-liner.
When there are multiple lines involved, it looks more confusing than readable.

Example of the same logic using just if condition.
```
relative_path = content_artifact.relative_path
if content_artifact.content.pulp_type == Package.get_pulp_type():
    relative_path = os.path.join (...)
```